.github/workflows/dependabot.yml

```diff
@@ -29,7 +29,6 @@
           PR_NUMBER: ${{ github.event.pull_request.number }}
           PR_URL: ${{ github.event.pull_request.html_url }}
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
-          JIRA_ASSIGNEE: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
```

Comment:
Does this action work for the mongocli-master branch? I see these issues:

1. As default, the GH action will use master instead of the mongocli-master branch to run the action, which means it will check the dependencies on master. 
2. the action checks out the master branch instead of mongocli-master (line 17)


I think you need to define a new action in master that run for pull request agains mongocli-master and checkout the mongocli-master branch

Reply:
which action implementation it will use? the one in `master` or the one in `mongocli-master`?  I think it will use the one in `master` 🤔 (I may be wrong, just make sure to double check)

Reply:
My understanding is that it's using the current branch one, since it is createing the action with "MongoCLI" instead of "AtlasCLI"